<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Manufacturing</class>
 <widget class="QMainWindow" name="Manufacturing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>534</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Departement de Production</string>
  </property>
  <property name="windowIcon">
   <iconset resource="rssources.qrc">
    <normaloff>:/images/logo_erp.png</normaloff>:/images/logo_erp.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="nbre_connectes">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>150</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="but_envoyer">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>142</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>** MES ***</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_gestion">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>711</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion</string>
    </property>
    <widget class="QPushButton" name="but_gestion_emp">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Affecter des Employes</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_gestion_sites">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Gerer les Sites</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_gestion_MP">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Gerer la MP</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_gestion_prod">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Production</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="logo_main_window">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>101</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/logo_edf_erp.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_suivi">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>471</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Suivi</string>
    </property>
    <widget class="QPushButton" name="but_control">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Control</string>
     </property>
    </widget>
    <widget class="QPushButton" name="but_repair">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>121</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>Reparation</string>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="logo_groupe">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>331</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/logo_groupe.png);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="nbre_connectes_2">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>150</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Utilisateurs connéctés</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>150</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Bienvenue, Mr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quitter_but">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>430</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Quitter</string>
    </property>
   </widget>
   <widget class="QLabel" name="login_name">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/images/inbox.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionEnvoyer_un_Mail"/>
    <addaction name="actionEnvoyer_un_Message"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionEnvoyer_un_Mail">
   <property name="icon">
    <iconset resource="rssources.qrc">
     <normaloff>:/images/email.png</normaloff>:/images/email.png</iconset>
   </property>
   <property name="text">
    <string>Envoyer un Mail</string>
   </property>
  </action>
  <action name="actionEnvoyer_un_Message">
   <property name="icon">
    <iconset resource="rssources.qrc">
     <normaloff>:/images/inbox.png</normaloff>:/images/inbox.png</iconset>
   </property>
   <property name="text">
    <string>Envoyer un Message</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rssources.qrc"/>
 </resources>
 <connections/>
</ui>
